### ISSUE TITLE
Show a better message when ffmpeg is not found

### ISSUE THREAD

**Issue by deluan** [deluan/navidrome#155]
Currently if you don't have `ffmpeg` installed/available in your system, the only message you get is something like:
```
ERRO[0006] Error importing MediaFolder                   error="error extracting metadata files" folder=/music
ERRO[0006] Errors while scanning media. Please check the logs 
```

A proper message saying `ffmpeg not found` would be more helpful.

Maybe this could be an ERROR level message displayed during initialization. Should be a simple check in the `initial_setup.go` file.

**Comment by deluan** [comment]
Unfortunately the previous PR did not solve this issue, as the error returned by `cmd.CombinedOutput()` is not related to whether the command was found in the system or not.

Maybe a better approach would be to check for the existence of `ffmpeg` in the path during initialization, with [exec.LookPath()](https://golang.org/pkg/os/exec/#LookPath), and logging an ERROR if it is not found

**Comment by jvoisin** [comment]
Should this raise a `panic`, since Navidrome is unable to scan nor transcode properly is ffmpeg isn't installed?

**Comment by deluan** [comment]
No sure, I rather have only a ERROR log, as it could cause issues in servers where Navidrome is configured to restart in case of failure and the logs from previous runs could be lost, leaving the user with no way to know what is happening.

**Comment by deluan** [comment]
Fixed on #160
